index.html

```diff
@@ -0,0 +1,253 @@
+                        <ul class="preview-buttons">
+                            <li class="html-btn"><button class="html-btn-1">HTML</button></li>
+                            <li class="css-btn"><button class="css-btn-1">Bootstrap</button></li>
+                            <li class="ruby-btn"><button class="ruby-btn-1">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+            </div>
+        </section>
+        <section class="section-4">
+            <article class="article-2">
+                <h1 class="about-me">About me</h1>
+                <p class="about-description">
+                    I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <button class="about-button">
+                    <h4 class="about-text">Get my resume</h4>
+                </button>
+            </article>
+            <div class="line-break"></div>
+            <ul class="article-3">
+                <li class="list-1"><img src="images/Rectangle 55.svg" alt="rect" class="rec-image">Language</li>
+                <li class="list">Javascript</li>
+                <li class="list">Ruby</li>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list-1"><img src="images/Rectangle 56.svg" alt="rect" class="rec-image">Frameworks</li>
+                <li class="list">React.js</li>
+                <li class="list">RSpec</li>
+                <li class="list">Capybara</li>
+                <li class="list-1"><img src="images/Rectangle 57.svg" alt="rect" class="rec-image">Skills</li>
+                <li class="list">Database Management</li>
+                <li class="list">Version Control</li>
+                <li class="list">CLI</li>
+                <li class="list">App Design</li>
+                <li class="list">Web Development</li>
+            </ul>
+        </section>
+        <section class="section-5">
+            <form action="https://formspree.io/f/{form_id}" method="post" class="form-sec">
+                <h3 class="contact-text">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h3>
+                <input type="type" name="name" placeholder="Full Name" required id="name-id" minlength="4" maxlength="30">
+                <div class="seperate-line"></div>
+                <input type="email" name="email" placeholder="Email address" id="email-id" required>
+                <div class="seperate-line"></div>
+                <textarea name="text-area" placeholder="Write me something" id="text-area" rows="5" cols="30" maxlength="500"></textarea>
```

Comment:
Kindly add the required attribute to your text area input field to ensure your users submit a message👍